operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -245,6 +245,7 @@ function buildWDTParams_MD5() {
          " but the secret does not have this key."
       exit 1
     else
+      # TBD code review comment: pass credentials via filename instead of risking putting them in an env var
```

Comment:
We can probably enhance WDT to take file.

Reply:
OK - I'll leave the comment here as a reminder.